test/functional/cmdLineTests/forceLazySymbolResolution/build.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0"?>
+
+<!--
+  Copyright (c) 2020, 2020 IBM Corp. and others
+
+  This program and the accompanying materials are made available under
+  the terms of the Eclipse Public License 2.0 which accompanies this
+  distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+  or the Apache License, Version 2.0 which accompanies this distribution and
+  is available at https://www.apache.org/licenses/LICENSE-2.0.
+
+  This Source Code may also be made available under the following
+  Secondary Licenses when the conditions for such availability set
+  forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+  General Public License, version 2 with the GNU Classpath
+  Exception [1] and GNU General Public License, version 2 with the
+  OpenJDK Assembly Exception [2].
+
+  [1] https://www.gnu.org/software/classpath/license.html
+  [2] http://openjdk.java.net/legal/assembly-exception.html
+
+  SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+<project name="cmdLineTests" default="build" basedir=".">
+	<taskdef resource="net/sf/antcontrib/antlib.xml" />
+	<description>
+		Build cmdLineTests forceLazySymbolResolution
+	</description>
+
+	<!-- set properties for this build -->
+	<property name="DEST" value="${BUILD_ROOT}/functional/cmdLineTests/forceLazySymbolResolution" />
+	<property name="src" location="." />
+
+	<target name="buildvm" description="build VM_Test.jar">
+		<subant target="build">
+			<property name="DEST" value="${DEST}"/>
+			<fileset dir="../../VM_Test/" includes="build.xml" />
+		</subant>
+	</target>
+
+	<target name="dist" depends="buildvm" description="generate the distribution">
+		<copy todir="${DEST}">
+			<fileset dir="${src}" includes="*.xml"/>
+		</copy>
+	</target>
+
+	<target name="build" >
```

Comment:
We need to add a clean target. Similar to other build.xml
In this clean target, we need to call https://github.com/eclipse/openj9/blob/3caadd433c8157b403f337ecd7a057ae188d8297/test/functional/VM_Test/build.xml#L195